index.d.ts

```diff
@@ -318,9 +318,19 @@ declare class Shopify {
     list: (params?: any) => Promise<Shopify.IPaginatedResult<Shopify.IEvent>>;
   };
   fulfillment: {
+    /**
+     * @deprecated
+     */
```

Comment:
It exists in the older versions, yes, but already in `2022-01` these were deprecated and the new endpoints existed; see https://shopify.dev/api/admin-rest/2022-01/resources/fulfillment#post-orders-order-id-fulfillments-fulfillment-id-cancel. IMHO since these are deprecated in all API versions currently supported, it would make sense to mark them deprecated in the library as well, but of course these tags can be dropped at maintainer’s discretion

Reply:
There are other endpoints beside these that are deprecated/removed. I think that the JSDoc `@deprecated` tag here is not very useful. The final user does not see it and it gives the developer the impression that the endpoint still exist and this is not always true. I think it is better to rely on Shopify's warnings and return values.